learn2track/models/gru_msp.py

```diff
@@ -128,9 +128,12 @@ def _get_sample(distribution_parameters, noise):
         mu = distribution_parameters[:, :3]
         sigma = distribution_parameters[:, 3:6]
 
-        sample = mu + noise * sigma
+        samples = mu + noise * sigma
 
-        return sample
+        return samples
+
+    def _get_max_component_samples(self, distribution_parameters):
+        return distribution_parameters[:, :3]
```

Comment:
Are those values represent the means? What is the shape of `distribution_parameters` ?